addons/website_sale/models/product.py

```diff
@@ -124,7 +126,10 @@ class ProductTemplate(models.Model):
     public_categ_ids = fields.Many2many('product.public.category', string='Website Product Category',
                                         help="The product will be available in each mentioned e-commerce category. Go to"
                                         "Shop > Customize and enable 'E-commerce categories' to view all e-commerce categories.")
-    product_image_ids = fields.One2many('product.image', 'product_tmpl_id', string='Images')
+
+    product_image_ids = fields.One2many('product.image', 'product_tmpl_id', string="Extra Product Images", copy=True)
```

Comment:
copy=True?

Reply:
idem for all copy=True, to double check

Reply:
As discussed, we just keep the copy on the template images. Also removed the useless copy from the product.image.